### DISCUSSION TITLE
/claim limit per group

### DISCUSSION THREAD

**Discussion by LightDevil96** [GriefPrevention/GriefPrevention#919] [Q&A]
How to limit number of claim lands on group
VIP: 1
PREMIUM: 2
....
Can help me?

**Comment by bigpresh** [comment]
There's the `MaximumNumberOfClaimsPerPlayer` setting, but I don't believe we have any way to override it based on group membership.

Could well be a good feature to be handled by  an addon, I'd think; a `ClaimCreatedEvent` is fired when a claim is about to be created, so an addon could catch that, check how many claims the player already has and how many they're entitled to, and cancel that event to prevent the claim creation if it would take them over their limit.

**Comment by RoboMWM** [comment]
this suggestion is a duplicate somewhere.
